index.bs

```diff
@@ -12410,6 +12453,24 @@ attribute is the concatenation of the class string of the namespace, ".",
 and the class string that the interface would otherwise have without this
 extended attribute.
 
+
+<div algorithm>
+
+    A [=platform object=] implementing an [=interface=] |interface|, in [=Realm=] |realm|,
```

Comment:
This needs to somehow say that *interface* is the [primary interface](https://heycam.github.io/webidl/#dfn-primary-interface), because a platform object can implement many different interfaces (i.e., inherited interfaces; so glad consequential interfaces are gone).